index.bs

```diff
@@ -310,14 +312,15 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     to certain restrictions, as specified in [[#createCredential]] and [[#getAssertion]].
 
 : <dfn>Public Key Credential</dfn>
-:: Generically, a credential is data one entity presents to another in order to authenticate the former's identity [[RFC4949]].
-    A WebAuthn <em>[=public key credential=]</em> is a <code>{ identifier, type }</code> pair identifying authentication
-    information established by the authenticator and the [RP], together, at [=registration=] time. The authentication
-    information consists of an asymmetric key pair, where the public key portion is returned to the [RP], which stores it in
-    conjunction with the present user's account. The authenticator maps the private key to the [RP]'s [=RP ID=] and stores
-    it. Subsequently, only that [RP], as identified by its [=RP ID=], is able to employ the [=public key credential=] in
-    [=authentication=] ceremonies, via the {{CredentialsContainer/get()}} method. The [=[RP]=] uses its copy of the stored public
-    key to verify the resultant [=Authentication Assertion=].
+:: Generically, a <em>credential</em> is data one entity presents to another in order to <em>authenticate</em> the former to the
+    latter [[RFC4949]]. A WebAuthn <em>[=public key credential=]</em> is a <code>{ identifier, type }</code> pair identifying
```

Comment:
This is less true now that WebAuthn subclasses Credential Manager's `Credential` type, which includes the RFC4949 "signed value" in addition to the RFC4949 "credential".

This and the previous sentence are also inconsistent. The {identifier, type} pair is *not* the "data one entity presents to another in order to authenticate the former to the latter"; that also has to include the signed challenge. (I favor moving away from RFC4949's definitions for reasons including that it has no term for the data actually used to authenticate.)